### PR TITLE
timoni: Add version 0.19.0

### LINKED ISSUE #5492: [Request]: timoni
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Non-GUI tool
- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 500 stars and/or 150 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

timoni

### Description

data aware/efficient alternative to helm kubernetes package manager 

### Homepage

https://timoni.sh/

### Download Link(s)

https://github.com/stefanprodan/timoni/releases

### Some Indication of Popularity/Repute

https://www.youtube.com/watch?v=bbE1BFCs548

https://www.youtube.com/watch?v=vIdJm2W21mI

https://medium.com/@stefanprodan/introducing-timoni-next-gen-package-manager-for-kubernetes-29df39683000

### PR DESCRIPTION
**Pull request by fcbry**
Closes #5492
First contribution, so be gentle, I wasn't 100% certain how to deal with checksums/git versioning, so I used the existing cue.json to get started. Let me know what you think. 

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/timoni.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "version": "3.14.0",
```
Comment (aliesbelik) [inline bucket/timoni.json:2]:
```suggestion
  "version": "0.19.0",
```

Actual version.

File: bucket/timoni.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "description": "The package manager for Kubernetes",
+  "homepage": "https://timoni.sh",
+  "license": "Apache-2.0",
+  "depends": "cue",
+  "architecture": {
+    "64bit": {
+      "url": "https://github.com/stefanprodan/timoni/releases/download/v0.19.0/timoni_0.19.0_windows_amd64.zip",
+      "hash": "ab037ecc679fcbdfcf62203752182480d6a09a13a7560081bd9bce181bd5664d",
+      "extract_dir": "windows-amd64"
```
Comment (aliesbelik) [inline bucket/timoni.json:11]:
```suggestion
      "hash": "ab037ecc679fcbdfcf62203752182480d6a09a13a7560081bd9bce181bd5664d"
```

File: bucket/timoni.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    }
+  },
+  "bin": "timoni.exe",
+  "checkver": {
+    "url": "https://github.com/stefanprodan/timoni/releases"
```
Comment (aliesbelik) [inline bucket/timoni.json:16]:
```suggestion
    "github": "https://github.com/stefanprodan/timoni"
```

File: bucket/timoni.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  "autoupdate": {
+    "architecture": {
+      "64bit": {
+        "url": "https://github.com/stefanprodan/timoni/releases/"
```
Comment (aliesbelik) [inline bucket/timoni.json:21]:
```suggestion
        "url": "https://github.com/stefanprodan/timoni/releases/download/v$version/timoni_$version_windows_amd64.zip"
```

File: bucket/timoni.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+      }
+    },
+    "hash": {
+      "url": "$baseurl/timoni_${version}_checksums.txt"
```
Comment (aliesbelik) [inline bucket/timoni.json:25]:
```suggestion
      "url": "$baseurl/timoni_$version_checksums.txt"
```

**Review by aliesbelik — CHANGES_REQUESTED** [review]
Please try locally first with proposed changes.
Optionally you could use tools from [bin](https://github.com/ScoopInstaller/Main/tree/master/bin) to format and validate manifest.

**Review by aliesbelik — APPROVED** [review]
Looks good to me.

**Review by tech189 — APPROVED** [review]

**Comment by tech189** [comment]
Thank you both!

**Pull request merged**